### PR TITLE
chore: import packages only once

### PR DESCRIPTION
**Pull request by testwill**
**How to categorize this PR?**

/area TODO
/kind TODO

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator
NONE
```

### CODE REVIEW

File: pkg/provider-local/charts/charts.go [hunk 1]
```diff
@@ -16,7 +16,6 @@ package charts
 
 import (
 	"embed"
-	_ "embed"
```
Comment (ialidzhikov) [inline pkg/provider-local/charts/charts.go:19]: The import is required. Revert this.

Comment (testwill) [reply pkg/provider-local/charts/charts.go:19]: embed has been imported

**Comment by shafeeqes** [comment]
This PR can be closed in favour of https://github.com/gardener/gardener/pull/8503

**Review by rfranzke — COMMENTED** [review]
/close
in favor of https://github.com/gardener/gardener/pull/8503

**Pull request closed without merging**